js/app.js

```diff
@@ -0,0 +1,107 @@
+console.log('yo')
+// combine objects, arrays, and functions
+const curbArray = [
+	characters: {
+		name1: 'jeff',
+		name2: 'larry',
+		name3: 'susie',
+		name4: 'cheryl',
+		name5: 'funkhouser'
+	}
+]
+console.log(curbArray.name5)
+
+
+
+
+let curbObj = {
+	funny: true,
+	characters: ['larry', 'jeff', 'cheryl', 'susie', 'funkhouser']
+};
+console.log(curbObj.characters[4])
+
+let curb2 = [
+	characters: ['larry', 'jeff', 'cheryl', 'susie', 'funkhouser']
+	seasons: ['season 1', 'season 2', 'season 3', 'season 4', 'season 5', 'season 6', 'season 7', 'season 8', 'season 9'  ]
+
```

Comment:
it looks like you were trying #5.  reminder: arrays are just comma delimited lists without property keys.  
so ... 
```
let curb2 = [
          ['larry',........], // important comma
          ['season1', 'season2'.....]
]
```

Reply:
or if you wanted an array of objects each of which had one property that was an array, you could do
```javascript
let curb2 = [
    { 
        characters: ['larry',........] 
     }, // important comma
    { 
        seasons: ['season1', 'season2'.....] 
    }
]
```